### ISSUE TITLE
Regrid operation broken with invert_yaxis=True

### ISSUE THREAD

**Issue by jlstevens** [ioam/holoviews#1999]
Here is the example without ``invert_yaxis=True ``:

![working](https://user-images.githubusercontent.com/890576/31508793-1ad3bba0-af77-11e7-80f9-1282ec695040.gif)

When zooming out you can see that regrid is working dynamically. When ``invert_yaxis=True`` dynamic updates stop working:

![broken](https://user-images.githubusercontent.com/890576/31508805-2b6e0e3e-af77-11e7-86f4-c2d031c38660.gif)

**Comment by jbednar** [comment]
Any errors on the JS console in the second case?

**Comment by philippjfr** [comment]
I've already noted that the fix is to un-invert the ranges in the bokeh RangeXYCallback, something like this in the ``_process_msg`` method will do it:

```python
x_range = self.plot.handles['x_range']
if isinstance(x_range, Range1d) and x_range.start > x_range.end:
    msg['x_range'] = msg['x_range'][::-1]
y_range = self.plot.handles['y_range']
if isinstance(y_range, Range1d) and y_range.start > y_range.end:
    msg['y_range'] = msg['y_range'][::-1]
```

**Comment by philippjfr** [comment]
This was fixed in #2023.
